src/GB2260.php

```diff
@@ -15,9 +15,11 @@ public static function getData()
 
     public static function parse($code)
     {
+        //@codeCoverageIgnoreStart
```

Comment:
By using code coverage we increase code quality, not pursuit 100%, please remove code coverage ignore annotations, we'll figure out a way to cover that later :)

And please [squash](https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit) commits in this pull request, then I'll merge it in a couple of days.